packages/embeds/embed-react/tsconfig.json

```diff
@@ -13,6 +13,7 @@
       "@calcom/embed-snippet": ["../embed-snippet/src"]
     }
   },
-  "include": ["src", "env.d.ts"],
-  "exclude": ["dist", "build", "node_modules", "test-cal.tsx"]
+  "include": ["**/*.ts","**/*.tsx", "env.d.ts"],
```

Comment:
Validate all .ts and .tsx files now. Some files like inline.tsx were outside src that are used as playground but they can be good for verifying the EMbed API types as well.